src/ReportBase.php

```diff
@@ -416,6 +416,17 @@ public function toHtml() : Response
         return response()->view('report-engine::base-web', $this->getConfig());
     }
 
+    public function rowContextActionsForConfig() : array
+    {
+        $return = [];
+
+        foreach ($this->rowContextActions() as $action) {
+            $return [] = $action->toArray();
+        }
+
+        return $return;
```

Comment:
Just a thought:
```suggestion
        return collect($this->rowContextActions())->toArray();
```

Then we just need to add `use Illuminate\Contracts\Support\Arrayable;` and implement it on the BaseAction class